internal/controller/state/dataplane/configuration.go

```diff
@@ -1192,6 +1193,8 @@ func convertAddresses(addresses []ngfAPIv1alpha2.RewriteClientIPAddress) []strin
 	return trustedAddresses
 }
 
+// buildLogging converts the API logging spec (currently singular LogFormat / AccessLog fields
+// in v1alpha2) into internal slice-based representation used by templates.
 func buildLogging(gateway *graph.Gateway) Logging {
 	logSettings := Logging{ErrorLevel: defaultErrorLogLevel}
```

Comment:
Can we not just use this `Logging` struct instead of defining a new `LoggingSettings` struct that contains similar information?